pkg/solana/config/config.go

```diff
@@ -194,3 +197,71 @@ func (c *config) MaxRetries() *uint {
 	}
 	return c.defaults.MaxRetries
 }
+
+type Chain struct {
+	BalancePollPeriod   *utils.Duration
+	ConfirmPollPeriod   *utils.Duration
+	OCR2CachePollPeriod *utils.Duration
+	OCR2CacheTTL        *utils.Duration
+	TxTimeout           *utils.Duration
+	TxRetryTimeout      *utils.Duration
+	TxConfirmTimeout    *utils.Duration
+	SkipPreflight       *bool
+	Commitment          *string
+	MaxRetries          *int64
+}
```

Comment:
hmmm what's the difference in usage between this struct and this one? https://github.com/smartcontractkit/chainlink-solana/blob/466d7d1795ac665c02cb382ae2a42c3951c7b40c/pkg/solana/db/db.go#L37

Reply:
I tried to reuse the old ones originally, but the db types weren't compatible with TOML.
Fortunately, the db types only need to live through the transition, and will be dropped later on.